<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>productView</class>
 <widget class="QWidget" name="productView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>983</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="mainTable">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>3</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionAdd_to_list">
   <property name="icon">
    <iconset>
     <normalon>icons/list-1.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Add to list...</string>
   </property>
   <property name="toolTip">
    <string>Add to list</string>
   </property>
  </action>
  <action name="actionReload">
   <property name="icon">
    <iconset>
     <normalon>icons/repeat.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Reload</string>
   </property>
   <property name="toolTip">
    <string>Reload</string>
   </property>
  </action>
  <action name="actionRemove_from_list">
   <property name="icon">
    <iconset>
     <normalon>icons/multiply-1.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Remove from list...</string>
   </property>
   <property name="toolTip">
    <string>Remove from list</string>
   </property>
  </action>
  <action name="actionDelete_list">
   <property name="icon">
    <iconset>
     <normalon>icons/multiply-1.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Delete list</string>
   </property>
   <property name="toolTip">
    <string>Delete list</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
